pkg/cmd/server/kubernetes/node/node_config_test.go

```diff
@@ -45,19 +46,19 @@ func TestKubeletDefaults(t *testing.T) {
 					CacheUnauthorizedTTL: metav1.Duration{Duration: 30 * time.Second},
 				},
 			},
-			CAdvisorPort:                4194, // disabled
-			VolumeStatsAggPeriod:        metav1.Duration{Duration: time.Minute},
-			CertDirectory:               "/var/run/kubernetes",
-			CgroupRoot:                  "",
-			CgroupDriver:                "cgroupfs",
-			ClusterDNS:                  nil, // overridden
-			ClusterDomain:               "",  // overridden
-			ContainerRuntime:            "docker",
-			Containerized:               false, // overridden based on OPENSHIFT_CONTAINERIZED
-			CPUCFSQuota:                 true,  // forced to true
-			DockerExecHandlerName:       "native",
-			DockerEndpoint:              "unix:///var/run/docker.sock",
-			DockerShimSocket:            "/var/run/dockershim.sock",
+			CAdvisorPort:          4194, // disabled
+			VolumeStatsAggPeriod:  metav1.Duration{Duration: time.Minute},
+			CertDirectory:         "/var/run/kubernetes",
+			CgroupRoot:            "",
+			CgroupDriver:          "cgroupfs",
+			ClusterDNS:            nil, // overridden
+			ClusterDomain:         "",  // overridden
+			ContainerRuntime:      "docker",
+			Containerized:         false, // overridden based on OPENSHIFT_CONTAINERIZED
+			CPUCFSQuota:           true,  // forced to true
+			DockerExecHandlerName: "native",
+			DockerEndpoint:        "unix:///var/run/docker.sock",
+
```

Comment:
@derekwaynecarr looks like kube no longer defaults this, do we need to continue defaulting to this value?